app/assets/v2/js/vue-components.js

```diff
@@ -650,7 +642,7 @@ Vue.component('project-card', {
             [[ project.summary | truncate(500) ]]
           </p>
           <div class="text-left">
-            <a :href="project_url" target="_blank" class="btn btn-sm btn-primary font-smaller-2">View Project</a>
+            <a :href="project.url_project_page" target="_blank" class="btn btn-sm btn-primary font-smaller-2">View Project</a>
             <a :href="project.bounty.url" class="btn btn-sm btn-outline-primary font-smaller-2">View Bounty</a>
```

Comment:
Since you added it, shouldn't this also be in the new formal? `project.url_bounty_page`

Reply:
Thanks for the explanation!